aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -48,7 +48,7 @@ class QLinearPackWeightInt8 final : public c10::OperatorKernel {
   at::Tensor operator()(at::Tensor weight, c10::optional<Tensor> bias) {
     TORCH_CHECK(
         weight.dim() == 2,
-        "The weight tensor for quantized::linear_prepack should"
+        "The weight tensor for quantized::linear_prepack (fbgemm) should"
```

Comment:
revert? - it's not a specific requirement for fbgemm :)

Reply:
Not a part of this diff now.